src/lib/store-cash/store-cash-init.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import type { Config } from '../../types';
+
+import { storeCashFpti } from './store-cash-fpti';
+
+export const storeCashInit = (config : Config) => {
+  storeCashFpti(config, {
+    fltp: 'store-cash',
+    es: 'connectionStarted',
+    website: 'muse',
+    feature: 'offer',
+    subfeature1: 'store-cash',
+    subfeature2: 'sdk',
```

Comment:
I'd double check here with Matt and GDS to see if we're allowed to alter any of these variables, because I'm not sure what they're filtering on.